### PR TITLE
[Durable Objects] Add Durable Objects dashboard

### PR DESCRIPTION
**Pull request by vy-ton**

### CODE REVIEW

File: content/durable-objects/reference/graphql-analytics.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
 ---
-title: GraphQL analytics
+title: Metrics and analytics
 pcx_content_type: concept
 weight: 16
 ---
 
-# GraphQL analytics
+# Metrics and analytics
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/reference/graphql-analytics.md:7]:
```suggestion
# Metrics and analytics

```

File: content/durable-objects/reference/graphql-analytics.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+Durable Objects expose analytics for Durable Object namespace-level and request-level metrics. 
+
+The metrics displayed in the [Cloudflare dashboard](https://dash.cloudflare.com/) charts are queried from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically](#query-via-the-graphql-api) via GraphQL or HTTP client.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/reference/graphql-analytics.md]:
```suggestion
The metrics displayed in the [Cloudflare dashboard](https://dash.cloudflare.com/) charts are queried from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically via GraphQL](#query-via-the-graphql-api) or HTTP client.
```

File: content/durable-objects/reference/graphql-analytics.md [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
+
+{{<Aside type="note" header="Durable Object namespace">}}
+A Durable Object namespace is a set of Durable Objects that can be addressed by name, backed by the same class. There is only one Durable Object namespace per class. A Durable Object namespace can contain any number of Durable Objects.
+{{</Aside>}}
+
+## View via the dashboard
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/reference/graphql-analytics.md]:
```suggestion
## View metrics and analytics via the dashboard
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/durable-objects/reference/graphql-analytics.md [hunk 4]
```diff
@@ -1,10 +1,32 @@
 ---
-title: GraphQL analytics
+title: Metrics and analytics
```
Comment (rozenmd) [inline content/durable-objects/reference/graphql-analytics.md]:
```suggestion
title: Metrics and GraphQL analytics
```
?

File: content/durable-objects/reference/graphql-analytics.md [hunk 5]
```diff
@@ -1,10 +1,32 @@
...
+
+Durable Objects expose analytics for Durable Object namespace-level and request-level metrics. 
+
+The metrics displayed in the [Cloudflare dashboard](https://dash.cloudflare.com/) charts are queried from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically via GraphQL](#query-via-the-graphql-api) or HTTP client.
+
+{{<Aside type="note" header="Durable Object namespace">}}
+A Durable Object namespace is a set of Durable Objects that can be addressed by name, backed by the same class. There is only one Durable Object namespace per class. A Durable Object namespace can contain any number of Durable Objects.
+{{</Aside>}}
+
+## View metrics and analytics via the dashboard
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/reference/graphql-analytics.md:17]:
```suggestion
## View metrics and GraphQL analytics via the dashboard
```

File: content/durable-objects/reference/graphql-analytics.md [hunk 6]
```diff
@@ -1,10 +1,32 @@
 ---
-title: GraphQL analytics
+title: Metrics and GraphQL analytics
 pcx_content_type: concept
 weight: 16
 ---
 
-# GraphQL analytics
+# Metrics and analytics
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/reference/graphql-analytics.md:7]:
```suggestion
# Metrics and GraphQL analytics
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**